src/client/app/components/unit/EditUnitModalComponent.tsx

```diff
@@ -396,9 +438,10 @@ export default function EditUnitModalComponent(props: EditUnitModalComponentProp
 							placeholder='Note'
 							onChange={e => handleStringChange(e)} />
 					</FormGroup>
-				</Container></ModalBody>
+				</Container>
+				</ModalBody>
```

Comment:
The open tags are both on one line but this change puts the end tags on two lines. I think together is okay since it will reduces indenting in this case. If you want to keep this change then the open tags need to be the same and the file needs reformatting (normally from VSC auto format).

Reply:
I put them back on one line.